backend/app/helpers/spree/admin/currency_helper.rb

```diff
@@ -1,12 +1,13 @@
 module Spree
   module Admin
-    module GeneralSettingsHelper
-      def currency_options
+    module CurrencyHelper
+      def currency_options(selected_value = nil)
+        selected_value ||= Spree::Config[:currency]
```

Comment:
What do you think about something like `def currency_options(currency = Spree::Config[:currency])` ?

Reply:
@kushniryb it will break if someone will pass a blank value (eg. rails form helpers), that's why I've moved it into the method itself